std/haxe/macro/Expr.hx

```diff
@@ -65,7 +70,7 @@ enum Constant {
 	/**
 		Represents a string literal.
 	**/
-	CString(s:String);
+	CString(s:String, ?kind:StringLiteralKind);
```

Comment:
I wonder though what is the meaning of `null` here?

Reply:
It defaults to double quotes. But yes that should be part of the documentation.